src/styles/main.scss

```diff
@@ -0,0 +1,89 @@
+$border-style: 1px solid black;
+$borde-size: 1px;
+$box-size: 100px;
+$calendar-size: calc(($box-size + 1px) * 7 - 1px);
+$margin-left: calc($calendar-size - ($box-size + 1px));
+$standard-color: #eee;
+$chose-color: #ffbfcb;
+$gap: 1px;
+$week: (
+  mon: 0,
+  tue: calc($box-size + $gap),
+  wed: calc($box-size + $gap) * 2,
+  thu: calc($box-size + $gap) * 3,
+  fri: calc($box-size + $gap) * 4,
+  sat: calc($box-size + $gap) * 5,
+  sun: calc($box-size + $gap) * 6,
```

Comment:
leave only numbers (0-6) and move calculations inside margin-left style so you don't need to repeat it